public/app/screens/main/navigation/style.scss

```diff
@@ -5,46 +5,45 @@
 
   @include icon-bar();
 
-  &.icon-bar {
-    background: transparent;
+  &.vertical {
+    width: 100%;
+  }
 
-    &.vertical {
-      width: 100%;
-    }
+  & * {
+    cursor: pointer;
   }
 
-  .cc_navigation__link {
+  &__link {
     text-align: left;
-    color: $default-gray;
     padding: 10px 0 10px 0;
 
     &-image {
-      width: 20px;
-      height: 20px;
-      margin: 0 20px 0 10px;
-      background-color: $default-gray;
       float: left;
+      width: $icon-bar-image-width;
+      height: $icon-bar-image-height;
+
+      margin: 0 20px 0 10px !important;
 
       &.devices {
-        mask-image: url('./img/devices.svg');
+        background-image: url('./img/devices.svg');
       }
+
       &.settings {
-        mask-image: url('./img/settings.svg');
+        background-image: url('./img/settings.svg');
```

Comment:
No it does not. 
`mask-image` will not work in any IE. 
Those svg images will be converted into a font, so that colors will be easily controllable.